app/src/main/java/co/kr/woowahan_banchan/data/datasource/local/order/OrderDataSourceImpl.kt

```diff
@@ -41,4 +41,19 @@ class OrderDataSourceImpl @Inject constructor(
                 orderDao.insertItem(item)
             }
         }
+
+    override fun getIncompleteItemCount(): Flow<Result<Int>> =
+        orderDao.getIncompleteItems()
+            .filterNotNull()
+            .map { Result.success(it.size) }
+            .catch {
+                emit(Result.failure(it))
```

Comment:
```suggestion
                emit(Result.failure(it.toErrorEntity))
```
이렇게 해야 ErrorEntity 타입으로 들어갈 듯 ?

Reply:
아~!